packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1358,6 +1359,11 @@ class TabBarView extends StatefulWidget {
   /// {@macro flutter.widgets.scrollable.dragStartBehavior}
   final DragStartBehavior dragStartBehavior;
 
+  /// The fraction of the viewport that each tabView should occupy.
```

Comment:
This looks like it is copied from PageController.viewportFraction? 

https://github.com/flutter/flutter/blob/37a1aaf8c11a2963f066427089d8dd3709a4a48d/packages/flutter/lib/src/widgets/page_view.dart#L147

Usually we template in this case to prevent duplication and to keep documentation consistent across the code base. Can you do that here?

This is an example of how you define a template:

https://github.com/flutter/flutter/blob/37a1aaf8c11a2963f066427089d8dd3709a4a48d/packages/flutter/lib/src/widgets/actions.dart#L634

And how you use it:

https://github.com/flutter/flutter/blob/37a1aaf8c11a2963f066427089d8dd3709a4a48d/packages/flutter/lib/src/widgets/actions.dart#L1102

Reply:
Hi, yes I did follow the pageView comment however I thought it made more sense to use tab than page in the same comment on the tabs widget, and couldn't find any option for the template to have a parameter. So wasn't able to template it

Reply:
I think using `page` is fine, since the documentation throughout this file mentions `page` many times. This should use a template.

Reply:
Alright! I've updated the PR to reflect this change